packages/hub/queries/email-card-drop-requests.ts

```diff
@@ -51,6 +54,28 @@ export default class EmailCardDropRequestsQueries {
       };
     });
   }
+
+  async claim({
+    ownerAddress,
+    verificationCode,
+  }: {
+    ownerAddress: string;
+    verificationCode: string;
+  }): Promise<EmailCardDropRequest> {
+    let db = await this.databaseManager.getClient();
+
+    let { rows } = await db.query(
+      `
+        UPDATE email_card_drop_requests
+        SET claimed_at = $1
+        WHERE owner_address = $2 AND verification_code = $3 AND claimed_at IS NULL
```

Comment:
What do we want to do about transaction hash?

Reply:
I think the `drop-card` task will set that afterward, this is sort of “optimistic” locking to prevent multiple claims.